src/components/JobPost/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Row, Col, /*Image,*/ ListGroup } from 'react-bootstrap'
+
+import JobPostRow from '../JobPostRow'
+import CustomSkill from '../CustomSkill'
+import icon from '../../media/user-img.svg'
+import CompanyIcon from '../CompanyIcon'
+import './JobPost.scss'
+
+type JobPostProps = {
+  title: string
+  jobDescription: string
+  seniority: string
+  startingDate: string
+  skills: any[]
+}
+
+const JobPost = ({
+  title,
+  jobDescription,
+  seniority,
+  startingDate,
+  skills,
+}: JobPostProps) => {
+  return (
+    <>
+      <Row className="job-post-card">
+        <Col md={2}>
+          <CompanyIcon icon={icon} />
+          {/* <Image 
+              src={`/${user.image}`} 
+              alt="company-image"
+              className="image" 
+              fluid /> */}
```

Comment:
You could implement conditional rendering for user image

Reply:
Okey. I'll do it when the upload image feature is finished.